protos/agent_services_pb.rb

```diff
@@ -0,0 +1,30 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# Source: agent.proto for package 'fastlane_c_i'
+# Original file comments:
+# *
+# proto definitions for auto-generated client, server, and messages to pass using grpc.
+#
+# if you make any changes to this file, run
+#   `rake proto`
+# to regenerate the ruby code.
+#
+
+require 'grpc'
+require 'agent_pb'
```

Comment:
Could you use double quotes for these two `require`s, please?

Reply:
Sorry, can't do anything about the code style in these files. They are auto-generated by `grpc`.

Reply:
Gotcha, are the files generated excluded from rubocop in the `.rubocop.yml`?